internal/provider/camunda_cluster_connector_secret_resource.go

```diff
@@ -0,0 +1,232 @@
+					validators.StringLengthBetweenValidator{Min: 1, Max: 50},
+					validators.StringNoSpacesValidator{},
+				},
+			},
+			"value": schema.StringAttribute{
+				MarkdownDescription: "The value of the connector secret",
+				Required:            true,
+				// Todo: Its actually also possible to update the secret value in-place. Not sure whether
+				// that just needs a different implementation or the API is not documented in the spec.
+				PlanModifiers: []planmodifier.String{stringplanmodifier.RequiresReplace()},
+				Sensitive:     true,
+			},
+		},
+	}
+}
+
+func (r *CamundaClusterConnectorSecretResource) Configure(ctx context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+	// Provider not yet configured
+	if req.ProviderData == nil {
+		return
+	}
+
+	provider, ok := req.ProviderData.(*CamundaCloudProvider)
+
+	if !ok {
+		resp.Diagnostics.AddError(
+			"Unexpected Resource Configure Type",
+			fmt.Sprintf("Expected *CamundaCloud client, got: %T. Please report this issue to the provider developers.", req.ProviderData),
+		)
+
+		return
+	}
+
+	r.provider = provider
+}
+
+func (r *CamundaClusterConnectorSecretResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	var data camundaClusterConnectorSecret
+
+	diags := req.Config.Get(ctx, &data)
+	resp.Diagnostics.Append(diags...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	newClusterConnectorSecretConfiguration := console.CreateSecretBody{
+		SecretName:  data.Name.ValueString(),
+		SecretValue: data.Value.ValueString(),
+	}
+
+	ctx = context.WithValue(ctx, console.ContextAccessToken, r.provider.accessToken)
+
+	inline, err := r.provider.client.ClustersApi.
+		CreateSecret(ctx, data.ClusterId.ValueString()).
+		CreateSecretBody(newClusterConnectorSecretConfiguration).
+		Execute()
+
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Unable to create cluster connector secret",
+			fmt.Sprintf("Unable to create cluster connector secret, got error: %s",
+				err.(*console.GenericOpenAPIError).Body()),
+		)
+		return
+	}
+
+	if inline.StatusCode >= 300 {
+		resp.Diagnostics.AddError(
+			"Unable to create cluster connector secret",
+			fmt.Sprintf("Unable to create cluster connector secret, got status: %s", inline.Status))
+		return
+	}
+
+	tflog.Info(ctx, "Camunda cluster connector secret created", map[string]interface{}{
+		"Name":      data.Name,
+		"ClusterId": data.ClusterId,
+	})
+
+	diags = resp.State.Set(ctx, &data)
+	resp.Diagnostics.Append(diags...)
+}
+
+func (r *CamundaClusterConnectorSecretResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	var data camundaClusterConnectorSecret
+
+	diags := req.State.Get(ctx, &data)
+	resp.Diagnostics.Append(diags...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	ctx = context.WithValue(ctx, console.ContextAccessToken, r.provider.accessToken)
+
+	if data.Name == types.StringValue("") {
+
+	}
```

Comment:
Actually, golangci works "correctly" and it reports an empty branch here, can you take a look?

Reply:
I meant the Github action (its failing with errors) and I cannot see the linting (scrolled over the laggy text).

I run it now locally and check the errors.

Reply:
Ahh! Shoot - my browser was always lagging and I have not seen this! Nice ❤️